README-ar.md

```diff
@@ -0,0 +1,323 @@
+<div style="direction: rtl; text-align: right">
+<p align="center">
+    <img alt="oss image" src="./imgs/zoss-logo.svg" height="200px" width="200px">
+</p>
+
+<p align="center">
+  <a href="https://twitter.com/intent/tweet?text=How%20to%20promote%20your%20open-source%20projects%20@ZenikaOSS&url=https://github.com/zenika-open-source/open-source-promotion-cheat-sheet&hashtags=OpenSource,CheatSheet">
+    <img alt="tweet" src="https://img.shields.io/twitter/url/https/twitter?label=Share%20on%20twitter&style=social" target="_blank" />
+  </a>
+</p>
+
+<h1 align="center">دليل نشر وتطوير المشاريع مفتوحة المصدر</h1>
+
+> يساعدك هذا المحتوى على تقديم مشروعك مفتوح المصدر بشكل أفضل وأسرع
+
+قائمة اللغات المتوفرة:
+
+- [العربية](./README-ar.md)
+- [English](./README.md)
+- [Català](./README-ca.md)
+- [Deutsch](./README-de.md)
+- [Español](./README-es.md)
+- [پارسی](./README-fa.md)
+- [Français](./README-fr.md)
+- [日本語](./README-jp.md)
+- [Português](./README-pt.md)
+- [Русский](./README-ru.md)
+- [සිංහල](./README-si.md)
+- [Türkçe](./README-tr.md)
+- [简体中文](./README-zh-cn.md)
+- [繁體中文](./README-zh-tw.md)
+
+A language is missing? Want to improve this cheat sheet? [Contributions are welcome](./CONTRIBUTING.md)!
+
+[نسخة PDF من هذا الدليل](./pdf/cheat-sheet.pdf)
+
+## 1. 🎢 المتطلبات الأساسية
+
+<br />
+
+<details>
+<summary>👌 تأكد من تطوير مشروعك بشكل كافٍ</summary>
+<p>
+
+> يجب أن يكون المشروع/البرنامج مستقرًا بدرجة كافية حتى لا يواجه المستخدمون مشكلة في استخدامه!
+
+</p>
+</details>
+
+<details>
+<summary>😎 اختر اسمًا جيدًا للمشروع</summary>
+<p>
+
+> اختر اسمًا للتطبيق يمكن للمستخدمين تذكره بسهولة
+
+</p>
+</details>
+
+<details>
+<summary>💅 اكتب ملف README نظيفًا وفعالًا للبرنامج</summary>
+<p>
+
+> ملف README هو أول ما يراه الزائرون ، لذا اجعله بسيطًا وجميلًا وسهل القراءة [إليك قائمة بأمثلة README] (https://github.com/matiassingers/awesome-readme).
+
+</p>
+</details>
+
+<details>
+<summary>💪 تسليط الضوء على نقاط القوة في المشروع</summary>
+<p>
+
+> تأكد من أن نقاط قوة المشروع أكثر وضوحا للزوار
```

Comment:
```suggestion
> حدد نقاط قوة المشروع وضوحا للزوار
```

translation more loyal to the English version

Reply:
Thanks for your lovely review 😀.